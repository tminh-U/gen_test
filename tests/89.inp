11
9 3 12 17 12 13 8 19 20 12 19 
9
6 7
4 10
2 7
4 11
7 8
10 10
10 11
8 11
10 10
