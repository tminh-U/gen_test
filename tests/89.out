3
27
22
42
4
2
4
8
2
